randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1657,7 +1659,9 @@ Extra - asset_id: collision_camera_020
   > Door to EMMI Zone West Exit Path
       All of the following:
           Can Slide
-          Flash Shift or Spider Magnet or Use Spin Boost
+          Any of the following:
+              Flash Shift or Spider Magnet or Use Spin Boost
+              Bomb and Morph Ball and Infinite Bomb Jump (Intermediate)
```

Comment:
Testing Cross Bombs here might be worthwhile, by placing it after jumping on the first ledge.